### PR TITLE
add shhclient package

### PR DESCRIPTION
**Pull request by adambabik**
I can't use this package from `ethereum/go-ethereum` because of type mismatches.

### CODE REVIEW

File: shhclient/client.go [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+package shhclient
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
```
Comment (dshulyak) [inline shhclient/client.go]: this import should be changed, otherwise, there will be same problem

**Comment by adambabik** [comment]
> my idea was that it is possible to use rpcclient with original types, from go-ethereum

You can use `rpc.Client` from go-ethereum, but not `sshclient.Client` because it requires Whisper from go-ethereum while we use one from status-im.

**Comment by dshulyak** [comment]
> You can use rpc.Client from go-ethereum, but not sshclient.Client because it requires Whisper from go-ethereum while we use one from status-im.

i mean that you can use original whisper import whenever it is needed

**Comment by adambabik** [comment]
I could import both Whisper packages but I think in this case it makes sense to fork this small package to avoid that.

**Pull request merged**